tests/conftest.py

```
@@ -0,0 +1,44 @@
+    if not os.environ.get("VERA_JS_COVERAGE"):
+        yield
+        return
+
+    cov_dir = tmp_path_factory.mktemp("v8-coverage")
+    os.environ["NODE_V8_COVERAGE"] = str(cov_dir)
+
+    yield
+
+    # Generate text report at session end.
+    if any(cov_dir.iterdir()):
+        subprocess.run(
```

Ruff (0.15.6)

[error] `subprocess` call: check for execution of untrusted input

(S603)